<launch>

  <node pkg="positions" type="positions" name="positions" output="screen">      
		<param name="positions_server_topic" value="/positions_from_server" />
		<param name="hostname" value="calcul07" />
		<param name="port_nb" value="10000" />
		<param name="duration_time" value="50" />
		<param name="persistance" value="0.5" />
  </node>
  
</launch>
